cmd/guaccollect/cmd/deps_dev.go

```diff
@@ -124,7 +147,7 @@ func validateDepsDevFlags(natsAddr string, csubAddr string, csubTls bool, csubTl
 }
 
 func init() {
-	set, err := cli.BuildFlags([]string{"retrieve-dependencies"})
+	set, err := cli.BuildFlags([]string{"retrieve-dependencies", "enable-prometheus", "prometheus-address"})
```

Comment:
should maybe be in the cli flags package instead?

Reply:
Here Brandon is referring to this package: https://github.com/guacsec/guac/blob/main/pkg/cli/store.go

Reply:
👍